bin/functions/make_plots.R

```diff
@@ -40,21 +43,25 @@ plot_pod_dist <- function(pod_df, pod_qs, in_dat, viewopt) {
     pod_df <- pod_df[,c("bs_index", "log10_dose")]
     colnames(pod_df)[2] <- "dose"
   }
-  pod_qs <- data.frame(q = names(pod_qs), pod = pod_qs)
-  
-  pod_qs$q <- paste0(pod_qs$q, " = ", round(pod_qs$pod, 4))
-  ggplot(pod_df, aes(x = dose)) + 
-    geom_density(fill = "goldenrod1", alpha = 0.5) + 
-    geom_vline(data = pod_qs, aes(xintercept = pod, linetype = q), color = "blue", alpha = 0.5, lwd = 1) +
-    labs(x = x_ax_lab,
-         title = "Distribution of POD Estimates") + 
-    scale_linetype_discrete(name = "Quantiles") + 
-    xlim(x_ax_lims) + 
-    my_theme
+  par(mar=c(5.1, 4.1, 4.1, 8.1), xpd = F)
+  d <- density(pod_df$dose)
+  plot(d,
+       panel.first = grid(),
+       xlab = x_ax_lab,
+       ylab = "Density",
+       xlim = x_ax_lims,
+       main = "Distribution of POD Estimates",
+       frame.plot = F)
+  box(lwd = 2, bty = "l")
+  polygon(d, col=rgb(1,0.75,0.15,0.5))
+  abline(v = pod_qs, lty = c("dotted", "solid", "dashed"), col = "blue", lwd = 2)
+  par(xpd = T)
+  legend("topright", legend = names(pod_qs), col = "blue", lty = c("dotted", "solid", "dashed"), title = "Quantiles", lwd = 2, inset = c(-0.15,0), cex = 0.75)
```

Comment:
Need to bring back the median and the credible intervals reporting either in the legend, as before, or someplace else. I don't see it in the code that was dropped, so I'm wondering if it was dropped in an earlier commit?